code/src/components/Pages/FirstScreen.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable linebreak-style */
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { WelcomePage } from './WelcomePage'
+import { GameScreen } from './GameScreen'
+
+export const FirstScreen = () => {
+  const userName = useSelector((store) => store.labyrinth.userName)
+
+  return (
+    <main>
+      {userName === '' ? <WelcomePage /> : <GameScreen />}
```

Comment:
A tidy and nice solution to make a ternary operator like this in a first screen component!